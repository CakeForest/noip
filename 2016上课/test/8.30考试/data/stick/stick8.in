19
6 7 5 8 3 8 3 4 7 8 6 6 6 2 7 6 3 8 8
61
6 3 8 5 1 5 7 1 6 3 3 3 1 8 2 4 7 4 7 5 4 4 3 1 6 1 6 8 8 5 4 5 7 3 1 7 8 7 7 5 1 2 7 2 1 8 5 7 4 3 4 7 6 6 7 3 6 4 3 6 8
6
2 7 1 3 5 8
0