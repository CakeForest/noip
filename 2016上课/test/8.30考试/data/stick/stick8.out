37
17
13